scipy/ndimage/interpolation.py

```diff
@@ -62,12 +63,24 @@ def spline_filter1d(input, order=3, axis=-1, output=numpy.float64):
     output : ndarray or dtype, optional
         The array in which to place the output, or the dtype of the returned
         array. Default is `numpy.float64`.
+    %(mode)s
 
     Returns
     -------
     spline_filter1d : ndarray
         The filtered input.
 
+    Notes
+    -----
+    All functions in `ndimage.interpolation` internally use a b-spline
```

Comment:
I'd just say that they  "do spline interpolation" rather than "use a b-spline representation". Then the filter1d function determines the b-spline coefficients needed for the interpolation.

Reply:
Done, PTAL.